code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -78,16 +84,37 @@
 	move_to_delay = 4
 	maxHealth = 400
 	health = 400
+	base_attack_cooldown = 12
+	ai_holder = /datum/ai_holder/simple_animal/ranged/alien/queen
 	natural_armor = list(
 		melee = ARMOR_MELEE_KNIVES,
 		laser	= ARMOR_LASER_HANDGUNS,
 		energy	= ARMOR_ENERGY_SMALL
 		)
 
 /obj/item/projectile/neurotox
-	damage = 30
+	damage = 25
 	icon_state = "toxin"
```

Comment:
Oddly, we have a fork here. Player xenos use /obj/item/projectile/energy/neurotoxin (which deals a much more modest 5 TOX + 5 WEAKEN) instead. 

I can only assume this is a separate type to let simple_animal xenos deal damage to simple_animals or something in lieu of paralysis?

Even if we want to have a separate type, I'd suggest we change damage_type to DAMAGE_TOXIN to make more goddamn sense. I BELIEVE this would also stop it breaking bones, but I may be getting lost in the spaghetti.

Reply:
Yeah, player xenos intentionally operate under different rules, partly as a holdover from the SCOM days, partially because player xenos have been so fucked up by Bay over the years. DAMAGE_TOXIN would function a little fucky with how toxin damage works in Baymed, but a switch to DAMAGE_BURN would probably work better here and would help with the limb breaking issue.

Reply:
Worth calling out that BURN has its own special rules, because God has abandoned this codebase long ago - most notably, _it creates hotspots_. So an alien splitting toxin could set the room on fire.